sys-cluster/charliecloud/charliecloud-0.20.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools optfeature python-single-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hpc/${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/hpc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~x86-linux"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="ch-grow doc examples +squashfs squashfuse"
+
+# Extensive test suite exists, but downloads container images
+# directly and via Docker and installs packages inside using apt/yum.
+# Additionally, clashes with portage namespacing and sandbox.
+RESTRICT="test"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	squashfs? ( sys-fs/squashfs-tools )
+	squashfuse? ( sys-fs/squashfuse )"
```

Comment:
RDEPEND-only USE flags are banned by policy, 
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

You should put these to `optfeature`.

Reply:
Thanks for pointing this out, I moved them! 
What I dislike about the policy is that it makes it impossible to express a strong recommendation (which also upstream mentions), in this case to install `squashfs` since noticeable functionality will be missing (that can be expressed via `+squashfs` in `IUSE`). 
But of course, I'll adhere to policy (and understand the reasoning) — and if GLEP 62 is implemented, I'll be very happy about that, since it will bring back this possibility :-).

Reply:
If you think it'd be worth using it in ~100 % of cases, you can make it an unconditional dependency. Let me know what are your thoughts.

Reply:
100 % is not the case, so I would indeed leave it as it is now: It is useful in >90 % for users who build containers with charliecloud, but not for users running containers only. 
So in short, I'd leave it as is ;-).